docs/design.md

```diff
@@ -1112,16 +1109,16 @@ particular, the maximum number of hops for gossipped information to take
 before reaching a node is given by `ceil(log(node count) / log(max
 fanout)) + 1`.
 
-# Key-prefix Accounting, Zones & Permissions
+# Key-prefix Accounting and Zones
```

Comment:
This section is all out of date too, but let's leave it for now. @jseldess is going to be revamping the README shortly.